boards/arm/stm32l562e_dk/doc/index.rst

```diff
@@ -184,6 +186,7 @@ Default Zephyr Peripheral Mapping:
 
 - USART_1 TX/RX : PA9/PA10
 - I2C_1 SCL/SDA : PB6/PB7
+- SPI_1 SCL/SDA : PB6/PB7 (BT SPI bus)
```

Comment:
Wrong SPI signals and pins

Reply:
Thanks for catching the typo. The new patch has been uploaded to fix this.